cvat/apps/cf/static/cf/js/dashboardPlugin.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/* global
+    showMessage:false
+    DashboardView:false
+*/
+
+//CF Entrypoint
+
+window.addEventListener('dashboardReady', () => {
+  	const createWorkstreamApiInputTextId = 'cfCreateApiInputText';
```

Comment:
Wrong indentation.

Reply:
@nmanovic I prematurely included the CF app I've been building and this readme was associated with it. It's gone now. Apologies!